docs/06-quality-control.md

```diff
@@ -1,3 +1,45 @@
 # Quality control
 
 When processing data, it is considered good practice (and is also very useful!) to inspect the intermediate output files/data created after every few steps. Here are some resources that can guide you through this process.
+
+###Let's start with introduction:
+
+
+##Introduction about quality assessment process
+##Quality checks included in this video:
+
+
+1. Looking for susceptibility and pathological artifacts
+
+2. Checking whether motion exceeds your lab's thresholds
+
+3. Keeping records to make sure you have the data that you should have. 
+??? example "Quality Checks for fMRI Data - Lecture on quality control for neuroimaging data, especially fMRI data"
+    -   [URL](URL: https://www.youtube.com/watch?v=fvv2dr3pT7I)
```

Comment:
```suggestion
    -   [URL](https://www.youtube.com/watch?v=fvv2dr3pT7I)
```
`    -   [URL](https://www.youtube.com/watch?v=fvv2dr3pT7I)` will render:

    -   [URL](https://www.youtube.com/watch?v=fvv2dr3pT7I)

with the good URL embedded :)